esp-hal/src/time.rs

```diff
@@ -10,7 +10,7 @@
 #[cfg_attr(esp32, doc = "36_558 years")]
 #[cfg_attr(esp32s2, doc = "7_311 years")]
 #[cfg_attr(not(any(esp32, esp32s2)), doc = "more than 7 years")]
-pub fn current_time() -> fugit::Instant<u64, 1, 1_000_000> {
+pub fn uptime() -> fugit::Instant<u64, 1, 1_000_000> {
```

Comment:
Does it still make sense for this to return `Instant`? `uptime` sounds like a duration to me.